repositories.bzl

```diff
@@ -194,12 +247,48 @@ def rules_nodejs():
         sha256 = "abcf497e89cfc1d09132adfcd8c07526d026e162ae2cb681dcb896046417ce91",
     )
 
-def rules_pkg_deps():
-    abseil_py()
-    six()
+
+# The @federation markers are an experiment in how to pick up dependency stanzas
+# from the rule sets. Each rule set should have a deps.bzl file. Inside those,
+# there will be the @federation markers. Those must be designed so they can
+# be extracted and dropped directly into this file, replacing the same stanza.
+
+# @federation: BEGIN @rules_pkg
+# TODO: ptr to where this was extracted from
+# https://github.com/bazelbuild/rules_pkg/pkg/deps.bzl
+def abseil_py():
```

Comment:
This function and the following ones are already in third_party_repositories.bzl